tyr-runner/src/main/java/org/jboss/tyr/whitelist/WhitelistProcessing.java

```diff
@@ -108,7 +118,7 @@ boolean addUserToUserList(String username) {
         for (String key : regexMap.keySet()) {
             Command command = CommandsLoader.getCommand(key);
             if (command != null) {
-                command.setCommandRegex(regexMap.get(key));
+                ((AbstractCommand) command).setRegex(regexMap.get(key));
```

Comment:
no need to cast here, just return AbstractCommand from CommandsLoader.getCommand directly